pages/api/id/[form]/[submission]/download.tsx

```diff
@@ -62,14 +62,13 @@ const handler = async (req: NextApiRequest, res: NextApiResponse, props: Middlew
 
       ExpressionAttributeValues: {
         ":formID": formID,
-        ":submissionID": submissionID,
+        ":submission": `NAME#${submissionName}`,
```

Comment:
I think the `QueryCommand` can be replaced by the `GetCommand` (get-item) in this scenario.

Reply:
Probably has the same cost but since we want to get one specific submission and we know where to find it.

Reply:
Good idea.  I'll swap it out tomorrow morning.